README.md

```diff
@@ -1,16 +1,16 @@
 # Work_Zone
 
-Work zone code and documentation for WZDx, iCone, etc.
+Work zone code and documentation for WZDx, iCone, etc. 
 
 | Build | Quality Gate | Code Coverage |
 | :---- | :----------: | ------------: |
-|       |             |               |
+|       |              |               |
 
 # WZDX Translator
 
 ## Project Description
 
-This project is an open source, proof of concept for Work Zone Data Exchange message creation from CDOT data to include iCone and Cotrip. The purpose of this tool is to  build a translator from Cotrip work zone data and iCone data to the WZDx message (3.1). The message will contain required data by WZDx that can be populated by iCone and other Cotrip Work Zone resources. Basically, in this project, we run a python script to parse data from the iCone xml file and COtrip data which generates WXDx 3.1 messages in a geojson format.
+This is an open source, proof of concept solution for translating work zone data in the form of COtrip/Salesforce, iCone, and NavJOY messages to the standardized WZDx 3.1 format. This project was developed for CDOT. A unique translator has been developed for each of these message types. These translators read in the source message, parse out specific fields, and generate a WZDx 3.1 message. For more information on these message formats and the data mappings between these messages and the WZDx format, see the [documentation](translator/docs). Sample files are located [here](translator/sample files). All these translators are built to run from the command line and from GCP cloud functions, hosted within the CDOT RTDH (real time data hub). For more information on cloud hosting, see [GCP_cloud_function](translator/GCP_cloud_function). 
```

Comment:
```suggestion
This is an open source, proof of concept solution for translating work zone data in the form of COtrip/Salesforce, iCone, and NavJOY messages to the standardized WZDx 3.1 format. This project was developed for CDOT. A unique translator has been developed for each of these message types. These translators read in the source message, parse out specific fields, and generate a WZDx 3.1 message. For more information on these message formats and the data mappings between these messages and the WZDx format, see the [documentation](translator/docs). Sample files are located [here](translator/sample%20files). All these translators are built to run from the command line and from GCP cloud functions, hosted within the CDOT RTDH (real time data hub). For more information on cloud hosting, see [GCP_cloud_function](translator/GCP_cloud_function). 
```

Having the space in "sample files" causes the link to not show up.

Reply:
Also, we're hosting these functions in a separate WZDX project and not in the RTDH right?

Reply:
good point

Reply:
Updated relative link and re-worded CDOT environment description